app/controller/settings.js

```diff
@@ -0,0 +1,22 @@
+var config = module.exports = {};
+
+var fs = require('fs');
+config.data = require('../config.json');
+
+config.update = function (req) {
+	// Update the config object
```

Comment:
You need to localize data then update the entire data object this should all be done via getters and setters! The write function needs to be a separate function as well that is called here